code/datums/traits/negative.dm

```diff
@@ -578,3 +578,52 @@
 	H.remove_language(/datum/language/common)
 	if(!H.can_speak_language(/datum/language/draconic) && !H.can_speak_language(/datum/language/machine))
 		H.grant_language(/datum/language/japanese)
+
+/datum/quirk/allergic
+	name = "Allergic Reaction"
+	desc = "You have had an allergic reaction to medicine in the past. Better stay away from it!"
+	value = -1
+	mob_trait = TRAIT_ALLERGIC
+	gain_text = "<span class='danger'>You remember your allergic reaction to a common medicine.</span>"
+	lose_text = "<span class='notice'>You no longer are allergic to medicine.</span>"
+	medical_record_text = "Patient has a severe allergic reaction to a commong medicine."
+//	var/allergy_list = list(/obj/item/reagent_containers/food/snacks/grown/apple,
+//							/obj/item/reagent_containers/food/snacks/grown/banana
+//							/obj/item/reagent_containers/food/snacks/grown/berries
+//							/obj/item/reagent_containers/food/snacks/grown/cherries
+//							/obj/item/reagent_containers/food/snacks/grown/grapes
+//							/obj/item/reagent_containers/food/snacks/grown/pineapple
+//							/obj/item/reagent_containers/food/snacks/grown/tomato
+							//Was going to make it pick if you allergic to eating a food, or a chem. too much work
```

Comment:
Well; if you gave up on this part of the project... remove this since it is non-functional code

Reply:
Its already removed locally ill commit if/when theres other changes to make

Reply:
Alright; get back to me when it happens I guess?